policyengine_us/variables/gov/states/ar/tax/income/ar_taxable_capital_gains.py

```diff
@@ -10,12 +10,22 @@ class ar_taxable_capital_gains(Variable):
     defined_for = StateCode.AR
 
     def formula(person, period, parameters):
-        long_term_capital_gains = person("long_term_capital_gains", period)
-        short_term_capital_gains = person("short_term_capital_gains", period)
+        net_lt_capital_gains = person("long_term_capital_gains", period)
+        net_st_capital_gains = person("short_term_capital_gains", period)
+        # Can be capital loss
+        net_cap_gain = net_lt_capital_gains - net_st_capital_gains
         p = parameters(
             period
         ).gov.states.ar.tax.income.gross_income.capital_gains
-        taxable_long_term_capital_gains = long_term_capital_gains * (
-            1 - p.exempt_rate
+        capped_net_cap_gain = min_(net_cap_gain, p.exempt.cap)
+        taxable_capped_net_cap_gain = capped_net_cap_gain * (1 - p.exempt.rate)
+        st_capital_gain = max_(0, net_st_capital_gains)
+        taxable_capital_gain = st_capital_gain + taxable_capped_net_cap_gain
+
+        capital_loss = taxable_capital_gain < 0
+        # Taxable capital loss is capped separately
+        filing_status = person.tax_unit("filing_status", period)
+        capped_capital_loss = max_(
```

Comment:
if cap shouldn't it be min?

Reply:
It caps negative gains to 1,500, I have added a multiplication by -1 to more accurately represent that case:
```
        capital_loss = taxable_capital_gain < 0
        # Taxable capital loss is capped separately
        filing_status = person.tax_unit("filing_status", period)
        capped_capital_loss = min_(
            (taxable_capital_gain * (-1)), p.loss_cap[filing_status]
        )
        return where(capital_loss, capped_capital_loss, taxable_capital_gain)

```